### ISSUE TITLE
Disable auto-link on ctrl-shift-v

### ISSUE THREAD

**Issue by kmaustral** [zolrath/obsidian-auto-link-title#39]
Sometimes the titles are not meaningful, so it would be good to have a simple way of undoing or preventing auto-linking. This is usually possible with shift-paste. It would be great to have that for this plugin.

**Comment by davecan** [comment]
Came here to request the same. I routinely paste links to PDFs and other resources that don't have titles and the plugin keeps embedding the markdown version of the link when it isn't necessary.

Would be great if we could have the following option:

`Trigger only on default paste command   [ ]`

**Comment by davecan** [comment]
I'm not familiar with the Obsidian plugin API but if this is the complete API it may be that the Obsidian devs don't expose the plain text paste event for public hook?

https://github.com/obsidianmd/obsidian-api/blob/master/obsidian.d.ts

**Comment by zolrath** [comment]
That's correct from what I can tell @davecan, the "editor-paste" command doesn't give me any information as to how the paste itself happened. I believe you can accomplish the desired behavior with the options currently available though!

1. Disable "Enhance Default Paste" in the Auto Link Title options.
2. Bind the "Auto Link Title: Paste URL and auto fetch title" hotkey to ctrl/cmd + v

At least on my Mac laptop I'm testing this on right now, this results in cmd + v triggering a markdown link and cmd + shift + v pasting a standard link.

**Comment by daeh** [comment]
I would also love this feature. On mac, cmd-shift-v is "paste as plain text". It would be fantastic to have that shortcut circumvent the auto-replacement of urls. Could also resolve #36

**Comment by electblake** [comment]
+1
